en_US/data-integration/rules.md

```diff
@@ -1,27 +1,49 @@
+The Rule Engine is EMQX's built-in data processing component, which works with [data bridges](./data-bridges.md) and can be used to help extract, filter, enrich, transform and store IoT data to accelerate application integration and business innovation. 
 
-![image](./assets/rules/data-integration-arch.png)
+<img src="./assets/rule-engine.png" alt="SQL-based IoT Rule Engine" style="zoom:30%;" />
 
-## Composition of Rules
+The Rule Engine is particularly useful for organizations that need to process and analyze data from multiple sources, such as IoT devices or other sensor networks. By using the Rule Engine, organizations can create rules that filter out irrelevant data, perform calculations, and trigger alerts or notifications based on specific events or conditions.
 
-The rules describe **data source**, **data processing process**, and **processing result destination**:
+This chapter provides an in-depth exploration of the Rule Engine and its capabilities. Readers will learn how the Rule Engine processes data in real time by defining rules, [using built-in SQL functions](./rule-sql-builtin-functions.md), and [customizing rules with SQL statements](./rule-sql-syntax.md). Additionally, readers will learn how to access and analyze data from multiple sources and leverage the power of [jq](./rule-sql-jq.md), a lightweight and flexible command-line JSON processor that is now supported with EMQX 5.x Rule Engine. 
 
-- **data source**: the data source of a rule can be a message or event, or an external data system. Rules specify the source of data through the FROM clause of SQL;
+## What is Rule Engine
 
-- **data processing**: rules describe the data processing process through SQL statements and functions. The WHERE clause of SQL is used to filter data, the SELECT clause and SQL function are used to extract and transform data;
+EMQX uses a powerful SQL-based IoT rule engine for data processing and integration, where the rules are partly specified with a domain-specific SQL-like language.
 
-- **processing result destination**: a rule can define one or more actions to process SQL output results. If the SQL execution passes, the rules will perform corresponding actions in sequence, such as storing the processing results in the database or republishing them to another MQTT topic.
+<img src="./assets/rules/data-integration-arch.png" alt="image" style="zoom:40%;" />
 
 
-![sql_process](./assets/sql_process.png)
 
-### Introduction to Rule SQL Statements
+This article discusses the composition of rules in a rule engine, which consists of data source, data transformation, and processing result destination. It also introduces the rule-engine language, which is similar to SQL and explains the different types of actions that can be taken in response to the rule engine's output. Finally, it provides some typical use cases for rules, such as action monitoring.
```

Comment:
in a rule engine -> in the rule engine

> This article discusses the composition of rules in a rule engine, which consists of data source, data transformation, and processing result destination.

This sentence is hard to read and understand.

> It also introduces the rule-engine language, which is similar to SQL and explains the different types of actions that can be taken in response to the rule engine's output. Finally, it provides some typical use cases for rules, such as action monitoring.

No, the actions are not described by the SQL language.

After the title, I think the reader would expect an explanation of what the rule engine is and not a description of what this article is about?

Reply:
I think I understand now that "explains" refer to the article and not the SQL-like language but I think this can be made more clearly. Perhaps breaking up the sentence to several smaller sentences would help?